WcaOnRails/config/database.yml

```diff
@@ -27,6 +36,13 @@ development:
     <<: *default
     database: wca_development
     replica: true
+  developer_dump:
+    <<: *default
+    database: wca_dump_developer
```

Comment:
why do these two not have database_tasks: false, but the other dump dbs all have it?

Reply:
Because in local development, it makes sense to be able to operate on the dump databases (for example, if you try to develop a new feature for the dev dump). On test and production environments, dropping, resetting, altering or otherwise migrating the temporary dump databases does not make sense.